### ISSUE TITLE
Use Zenpy to Count the total number of open tickets with a tag

### ISSUE THREAD

**Issue by akhil007garg** [facetoe/zenpy#188]
Hi All,

I am new to python and I am trying to use Zenpy to count all the tickets in my Zendesk instance which are open and have a tags='abc'. Can anyone suggest how can this be done?

So for example - I have 5 open tickets with tags='abc'. Now I want Zenpy to search all the tickets and give me the total number.

sample code-
for ticket in zenpy_client.count(status='open', tags='abc'):
...     print(ticket.count)

"I know this query is wrong but its just an example"

Thank you in advance.

**Comment by facetoe** [comment]
Hi @akhil007garg,

You should be able to do this as follows (untested):

```python
print(len(zenpy_client.search(type='ticket', tags='abc')))
```

**Comment by akhil007garg** [comment]
Thank you for a quick reply facetoe. I think, I found a solution which will give me an exact count.

What I did was
1. I imported the class which holds count function
2. I created a variable holding my search string for example - tick = zenpy_client.search(type='ticket', status='pending', tags='abc')
3. Called this variable with function count - tick.count

Hence, it gave me the exact number I needed.

Thank you
